src/components/Button/Button.js

```diff
@@ -0,0 +1,7 @@
+import styles from './Button.module.css';
+
+function Button({ innerButtonText }) {
```

Comment:
u can add children props and use button with this prop
 <Button>Text</Button>

```suggestion
function Button({ children }) {
```